gems/aws-sdk-core/spec/aws/credential_resolution_chain_spec.rb

```diff
@@ -91,11 +100,17 @@ module Aws
           'AR_TOKEN'
         )
         client = ApiHelper.sample_rest_xml::Client.new(
-          profile: 'ar_plus_creds', region: 'us-east-1'
+          profile: 'ar_plus_creds', region: 'us-east-2'
```

Comment:
Is this trying to demonstrate the two regions? What about the classical us-west-2?

Reply:
Yeah - was just making sure it was passing something different.  I'll change to us-west-2 for tradition.